### DISCUSSION TITLE
Dated School ID

### DISCUSSION THREAD

**Discussion by lSamWorsnopl** [community/community#42536] [GitHub Education]
Currently it is asking me to re-verify academic status but I do not have a dated university ID so I am giving other proofs it shows but it always gives me the same message that they were not accepted. 
![321494216_1385538168857441_1681143239503034570_n](https://user-images.githubusercontent.com/114540198/209418361-e3fa11d5-af21-400e-8ce7-dfeeae9c4dcb.jpg)
It shows all my data and also the current date so I don't understand the reason why they are not accepted. Also, it no longer gives me the option to upload a file, only to take a picture.

**Comment by SilasRodrigues19** [comment]
Hi @lSamWorsnopl 

There is a similar discussion that was resolved, at the following link:

[https://github.com/community/community/discussions/41657](https://github.com/community/community/discussions/41657)

Hope this helps you.

**Comment by lSamWorsnopl** [comment]
Although it does not allow me to upload screenshots like the one in this publication.

**Comment by kubraevren** [comment]
Merhaba, öğrenci belgemin ekran görüntüsünü sisteme yükleyince kabul etti.
öğrenci belgesini png olarak yükle. ve oldu.
